src/components/Navigation.css

```diff
@@ -9,4 +9,5 @@
 
 .Navigation__icon {
   color: black;
```

Comment:
Do you have a file with your colors defined already? i'd recommend it so you can change them later, and also so practice css variables. You could do that in a separate branch though